jwst/ami/NRM_Model.py

```diff
@@ -11,13 +18,15 @@
 # - updated May 2013 to include hexagonal envelope
 # - updated (hard refactored) Oct-Nov 2014 Anand S.
 
-from __future__ import absolute_import, division
+from __future__ import (absolute_import, division)
```

Comment:
Do we still need this now that we're no longer supporting py2.7? Are there places in the remaining code that aren't yet updated to py3.5?